deploy/cluster-monitoring-config/cluster-monitoring-config.yaml

```
@@ -12,14 +12,14 @@ data:
         - effect: NoSchedule
           key: node-role.kubernetes.io/infra
           operator: Exists
-      retention: 7d
+      retention: 15d
```

With the bump in both storage and retention we are not gaining any headroom on consuming all storage for prometheus, which is the goal with OSD-4875.  Recommend we do not make this change in retention and leave the PR to only the storage change.

```suggestion
      retention: 7d
```

---

I think @jewzaam suggests to change both, just not in the same PR/promotion

---

I don't see why we need the retention.  What is the driver to increase retention?

---

retention has been reduced from 10 to 7 days as a short-term solution to get rid of the alerts. Now we want to increase the retention instead as 7 days of metrics is a short period. With OSD-5221 we did a spike to figure out the right way to go forward with this, got approval for enough storage to hold 15 days, and that's what @georgettica is implementing here. Current config has enough headroom for 7 days, so the assumption is the higher amount should be sufficient for 15 days.

---

Thought we were sitll hitting storage problems with 7d retention and 50Gi storage, so I disagree that this bump to 14d retention and 100Gi storage fixes the storage issue.  If we want to bump to 10d I am OK with that.

---

What size would be okay on your end? You can see the calculations in the spike we did and devise from there.
I would like to work on both avenues (bigger storage for 14 days or current suggestion for 10) in case one gets blocked (by BU or internally)

---

Without a use case driving 14d over 10d I'm +1 to 100Gi w/ 10d.  Having over a week is nice and I have felt 10d was a reasonable number after we dropped it from 14d originally.

---

I think I commented on the ticket: 2 weeks is the minimum that allows you to do WoW for any day in the last seven days.
For example if you need to compare the behavior of a cluster between two weeks.
As long as we don't federate all metrics off cluster, I think we shouldn't lose that ability on cluster, hence the 14d

---

I read back through OSD-5221 reasoning.  And considering a low volume of KubePersistentVolumeFullInFourDays alerts showing in Tableau, no hits in telemeter in the recent past, inability to get a result from telemeter for longer time slices (just get a timeout for queries), we're probably OK with the 7d and we can adjust retention down again if necessarily I'll approve.